src/Cms/Translation.php

```diff
@@ -83,6 +84,26 @@ public function direction(): string
 		return $this->get('translation.direction', 'ltr');
 	}
 
+	/**
+	 * Returns the first day of the week (0 = Sunday ... 6 = Saturday)
+	 * for the translation's locale (or as defined via config option)
+	 *
+	 * @since 4.5.0
+	 */
+	public function firstWeekday(): int
+	{
+		$weekday = App::instance()->option('date.weekday');
+
+		if (is_int($weekday) === true) {
+			return $weekday;
+		}
+
+		$locale   = $this->locale();
+		$calendar = IntlCalendar::createInstance(null, $locale);
+
+		return $calendar->getFirstDayOfWeek() - 1;
```

Comment:
This enhancement requires `intl` php extension even `date.handler` haven't set `intl`. I think this is a breaking change for users who not installed `intl` php extension. 

I have a solution for this issue. Roughly we can check the date handler or check the `intl` extension. I'm not sure which one is best way but checking `date.handler` seems more consist.

```suggestion
        $kirby   = App::instance();
		$weekday = $kirby->option('date.weekday');

		if (is_int($weekday) === true) {
			return $weekday;
		}
		
		// returns sunday as first day of week if date handler is not `intl`
		if ($kirby->option('date.handler') !== 'intl') {
		    return 0;
		}

		$locale   = $this->locale();
		$calendar = IntlCalendar::createInstance(null, $locale);

		return $calendar->getFirstDayOfWeek() - 1;
```